README.md

```
@@ -127,17 +127,18 @@ To save sensor data for future analysis, follow these steps:
-| Application                | Description                                                      |  
-|----------------------------|------------------------------------------------------------------|  
-| **ECG with Heart Rate**    | Real-time ECG with BPM calculation.                              | 
-| **EMG with Envelope**      | Real-time EMG Visualization with Envelope.                       |
-| **EOG with Blinks**        | Real-time EOG Signal visualization with Blinks marked as Red Dot.|
-| **EEG with FFT**           | Real-time EEG Signal visualization with FFT and Brainpower bands.|
-| **EEG Tug of War Game**    | 2 Player EEG Based Game                                          |
-| **EEG Beetle game**        | Real-time EEG focus based game.                                  |
-| **EOG Keystroke Emulator** | Blink detection triggers spacebar.                               |  
-| **GUI**                    | Visualize raw data in real-time                                  |
-| **CSV Plotter**            | Tool to plot the recorded CSV Files                              |
+| Application                | Description                                                        |  
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)